src/Symfony/Component/Yaml/Parser.php

```diff
@@ -912,4 +1000,21 @@ private function isBlockScalarHeader()
     {
         return (bool) preg_match('~'.self::BLOCK_SCALAR_HEADER_PATTERN.'$~', $this->currentLine);
     }
+
+    private function getLineTag($value, $nextLineCheck = true)
+    {
+        if ('' === $value || '!' !== $value[0] || 1 !== preg_match('/^'.self::TAG_PATTERN.' *( +#.*)?$/', $value, $matches)) {
+            return;
+        }
+
+        if ($nextLineCheck && !$this->isNextLineIndented()) {
+            return;
+        }
+
+        if (!isset($this->tags[$matches['tag']])) {
+            throw new UnsupportedTagException($matches['tag']);
```

Comment:
In a previous comment, you noted that this could be a BC break. Should we trigger a deprecation instead? (same at other places where this is thrown)